packages/constants/src/logLevels.js

```diff
@@ -0,0 +1,7 @@
+// Note: Winston log levels are lower case, so we match that here
```

Comment:
WAIT I just noticed this. If log levels are lower-cased, why are we testing for camel-cased in the tests?

Reply:
oh, I think winston log levels are camelCase (just they're all one word so whatever)